.github/workflows/coverage.yaml

```diff
@@ -48,8 +51,14 @@ jobs:
   comment:
     runs-on: ubuntu-latest
     needs: [coverage]
+    permissions:
+      pull-requests: write
```

Comment:
## GITHUB_TOKEN Permissions
### Original permissions in non paranoid repositories (when not dealing with forks)
```yaml
  Actions: write
  Checks: write
  Contents: write
  Deployments: write
  Discussions: write
  Issues: write
  Metadata: read
  Packages: write
  Pages: write
  PullRequests: write
  RepositoryProjects: write
  SecurityEvents: write
  Statuses: write
```

### Original permissions in non paranoid repositories (when dealing with forks)
```yaml
  Actions: read
  Checks: read
  Contents: read
  Deployments: read
  Discussions: read
  Issues: read
  Metadata: read
  Packages: read
  Pages: read
  PullRequests: read
  RepositoryProjects: read
  SecurityEvents: read
  Statuses: read
```

### Original permissions in paranoid repositories
```yaml
  Contents: read
  Metadata: read
  Packages: read
```

### Reduced Permissions (when not dealing with forks)
```yaml
  Pull-Requests: write
  Metadata: read
```

### Reduced Permissions (when dealing with forks)
```yaml
  Pull-Requests: read
  Metadata: read
```

Note that this isn't good enough to result in things working... to make it work requires that additional bit of stuff...